on-new-fortress.lua

```diff
@@ -4,9 +4,9 @@ local HELP = [====[
 
 on-new-fortress
 ===============
-Runs commands like ``multicmd``, but only in a newborn fortress.
+Runs commands like :ref:`multicmd`, but only in a newborn fortress.
 
-Use this in ``onMapLoad.init`` with f.e. ``ban-cooking``::
+Use this in ``onMapLoad.init`` with f.e. :ref:`ban-cooking`::
```

Comment:
You don't actually need the `:ref:` for links to other docs (in fact, I didn't know that worked). Just single backticks will produce links in RST (double backticks are code markup)

Reply:
reST is new to me and so far strange.